security/src/test/java/org/kaazing/gateway/security/auth/context/DefaultLoginContextFactoryTest.java

```diff
@@ -126,5 +129,26 @@ public void testBasicAuthCustomLoginModuleLoginSuccess() throws Exception {
         assertNotNull(loginContext);
         loginContext.login();
     }
+
+    @Test
+    public void testInetAddressFilterCustomLoginModuleLoginSuccess() throws Exception {
+        InetAddress address = InetAddress.getByName("localhost");
+        context.checking(new Expectations() {
+            {
+                oneOf(configuration).getAppConfigurationEntry(REALM_NAME);
+                final String loginModuleName = "org.kaazing.gateway.security.auth.SimpleInetAddressTestLoginModule";
```

Comment:
Better to use `SimpleInetAddressTestLoginModule.class.getName()` instead to survive a refactor?